share/completions/eopkg.fish

```diff
@@ -0,0 +1,153 @@
+#
+# Completions for the eopkg command
+#
+
+# Frequently used eopkg commands
+
+# Eopkg additional completion test
+function __fish_eopkg_package_ok -d 'Test if eopkg should have packages as completion candidate'
+    for i in (commandline -poc)
+        if contains -- $i upgrade up remove rm install it info check
```

Comment:
This entire function can be simplified to  `complete -c eopkg -n '__fish_seen_subcommand_from upgrade up remove rm #...' -a '(__fish_print_packages)'`.